swap/peer.go

```diff
@@ -69,10 +69,10 @@ func (sp *Peer) handleMsg(ctx context.Context, msg interface{}) error {
 // TODO: this should not be blocking
 func (sp *Peer) handleEmitChequeMsg(ctx context.Context, msg *EmitChequeMsg) error {
 	cheque := msg.Cheque
-	log.Debug("received emit cheque message from peer", "peer", sp.ID().String())
+	log.Info("received cheque from peer, attempting to submit and cash", "peer", sp.ID().String())
```

Comment:
The idea here is good: important stuff related to `Swap`, which could be viewed like a financial audit, should be logged prominently ("received cheque from peer"). There is an issue for this: https://github.com/ethersphere/swarm/issues/1593

Nevertheless, this log is "polluted". It also has debugging information: "attempting to submit and cash". An audit log would be "Cheque submitted", or "Cheque cashed", or "Cheque rebounced" ("business" information).

In order to not have two logs, I suggest you keep `.Info` but only write `"received cheque from peer"`

Reply:
You didn't address this point, I will merge and then address in a different commit